app/config.js

```diff
@@ -20,6 +20,10 @@ var blinken_config = {
           "name": "CI",
           "url": "https://ci-alert.integration.publishing.service.gov.uk"
         }
+        {
+          "name": "AWS Staging"
+          "url": "https://alert.staging.govuk.digital"
```

Comment:
Ah, I thought these `govuk.digital` domains weren't externally routed? But anyway, it works :)

Reply:
They are, but restricted to office IPs and we haven't configured the auto-redirect from HTTP => HTTPS.